zephyr/Kconfig

```diff
@@ -56,4 +56,11 @@ config ZEPHYR_DP_SCHEDULER
 	  DP modules can be located in dieffrent cores than LL pipeline modules, may have
 	  different tick (i.e. 300ms for speech reccognition, etc.)
 
+config INLINE_CONSIDER_ALL_FUNCTIONS
+	bool
+	default y if CAVS_VERSION_2_5 || ACE
+	default n
+	help
+	  Indicate whether noinine_functions are added into build option.
+
```

Comment:
Sorry, missed this in first review, but please don't define this twice if this is used both for XTOS and Zephyr, but rather put at top-level Kconfig.sof. See e.g. option COMPILER_WORKAROUND_CACHE_ATTR . You could put this next to that one.